src/runtime/hexagon_mini_dma.h

```diff
@@ -0,0 +1,26 @@
+#ifndef _MINIDMA_H
+#define _MINIDMA_H
+
+/*!
+ * Format IDs
+ */
+typedef enum {

```

Comment:
Please fix this enum to use small case. The enum values can themselves by in block capitals. But, camelcase is not good.